diff --git a/node_modules/@twa-dev/sdk/dist/telegram-web-apps.js b/node_modules/@twa-dev/sdk/dist/telegram-web-apps.js
index c399b5f..f682c7e 100644
--- a/node_modules/@twa-dev/sdk/dist/telegram-web-apps.js
+++ b/node_modules/@twa-dev/sdk/dist/telegram-web-apps.js
@@ -1,6 +1,7 @@
 "use strict";
 // WebView
 (function () {
+    return;
     var eventHandlers = {};
     var locationHash = '';
     try {
@@ -776,7 +777,7 @@
                 return;
             }
             curButtonState = btn_state;
-            WebView.postEvent('web_app_setup_main_button', false, btn_params);
+            // WebView.postEvent('web_app_setup_main_button', false, btn_params);
             if (initParams.tgWebAppDebug) {
                 updateDebugButton(btn_params);
             }
